resources/zib/zib-Encounter.xml

```diff
@@ -0,0 +1,237 @@
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/PractitionerRole" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/RelatedPerson" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-HealthProfessional-Practitioner" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-HealthProfessional-PractitionerRole" />
+      </type>
+      <mapping>
+        <identity value="zib-encounter-v4.0.1-2020EN" />
+        <map value="NL-CM:15.1.7" />
+        <comment value="ContactWith" />
+      </mapping>
+    </element>
+    <element id="Encounter.period.start">
+      <path value="Encounter.period.start" />
+      <short value="StartDateTime" />
+      <definition value="The date and time at which the contact took or will take place." />
+      <alias value="BeginDatumTijd" />
+      <mapping>
+        <identity value="zib-encounter-v4.0.1-2020EN" />
+        <map value="NL-CM:15.1.3" />
+        <comment value="StartDateTime" />
+      </mapping>
+    </element>
+    <element id="Encounter.period.end">
+      <path value="Encounter.period.end" />
+      <short value="EndDateTime" />
+      <definition value="The date and time at which the contact ended or will end. If the contact takes place over a period of time, this indicates the end of the period, in the case of an admission, for example." />
+      <alias value="EindDatumTijd" />
+      <mapping>
+        <identity value="zib-encounter-v4.0.1-2020EN" />
+        <map value="NL-CM:15.1.4" />
+        <comment value="EndDateTime" />
+      </mapping>
+    </element>
+    <element id="Encounter.reasonCode.text">
+      <path value="Encounter.reasonCode.text" />
+      <short value="DeviatingResult" />
+      <definition value="A deviating result which serves as the reason for the contact." />
+      <alias value="AfwijkendeUitslag" />
+      <mapping>
+        <identity value="zib-encounter-v4.0.1-2020EN" />
+        <map value="NL-CM:15.1.12" />
+        <comment value="DeviatingResult" />
+      </mapping>
+    </element>
+    <element id="Encounter.reasonCode.text.extension">
+      <path value="Encounter.reasonCode.text.extension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Encounter.reasonCode.text.extension:commentContactReason">
+      <path value="Encounter.reasonCode.text.extension" />
+      <sliceName value="commentContactReason" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-Encounter.CommentContactReason" />
+      </type>
+    </element>
+    <element id="Encounter.reasonReference">
+      <path value="Encounter.reasonReference" />
+      <short value="Problem, Procedure" />
```

Comment:
Geen comma: 
"If there are multiple zib elements mapped to a single FHIR element, it will be formatted like ZibConcept1 / ZibConcept2 (unless the names are the same)."

Reply:
Fixed with a slash